pkg/ytconfig/canondata/TestGetStrawberryControllerConfig/test.canondata

```diff
@@ -11,7 +11,9 @@
         chyt={
             "address_resolver"={
                 "enable_ipv4"=%true;
-                "enable_ipv6"=%true;
+                "enable_ipv6"=%false;
```

Comment:
cause of
- https://github.com/ytsaurus/ytsaurus-k8s-operator/pull/324/files#diff-9875b847729a36cdf9f8ee8333cefa406ed3e7fb1f5f297c87e5e3e556d907bbR558
- https://github.com/ytsaurus/ytsaurus-k8s-operator/pull/324/files#diff-ee85a92ac74a0dc4dbb84dab8829fe475be259f340407f523529d5dda26961ddR315-R317

Reply:
So it was hardcoded for strawberry, but now it is configured according to the common value, ok

Reply:
On a second though that may be backward incompatible change and maybe we need to think here more.

Reply:
So users, who have `ipv4=false;ipv6=true` or `ipv4=true;ipv6=false` (assume false/false is invalid configuration) will redeploy strawberry on operator update with this settings, not true/true as before which can break things for them. Not sure yet how problematic such update could be.

Reply:
I suppose we can describe breaking changes in the next release about strawberry ipv4/6.